src/components/attendee/NavOverlay/ScheduleOverlay/ScheduleOverlay.tsx

```diff
@@ -69,116 +82,163 @@ export const ScheduleOverlay: React.FC = () => {
+        ? currentVenueBookMarkEvents
+        : currentVenueEvents,
+    };
+  }, [
+    firstScheduleDate,
+    selectedDayIndex,
+    liveAndFutureEvents,
+    showPersonalisedSchedule,
+    space,
+  ]);
+
   const renderedEvents = useMemo(
     () =>
-      weekdays.map(({ scheduleDate, daysWithEvents }) => (
-        <div key={scheduleDate}>
-          <div>{scheduleDate}</div>
-          {daysWithEvents.map((event) => {
-            const isCurrentEventLive = isEventLive(event);
-            const showDate = Boolean(
-              differenceInCalendarDays(
-                eventEndTime({ event }),
-                eventStartTime({ event })
-              )
-            );
-
-            return (
-              <div key={event.id}>
-                <div>{event.name}</div>
-                <div>
-                  <span>
-                    {isCurrentEventLive
-                      ? "Live "
-                      : formatTimeLocalised(eventStartTime({ event })) + " - "}
-                  </span>
-                  <span>{isCurrentEventLive && "until "}</span>
-                  <span>
-                    {showDate &&
-                      formatDateRelativeToNow(eventEndTime({ event }))}
-                  </span>
-                  {STRING_SPACE}
-                  <span>{formatTimeLocalised(eventEndTime({ event }))}</span>
-                  <span> in {event.spaceId}</span>
-                </div>
-                <div>{event.description}</div>
-                <div
-                  className={CN.scheduleOverlayBookmark}
-                  onClick={(e) => handleEventBookmark(e, event)}
-                >
-                  {event.isSaved
-                    ? "Removed from bookmarked"
-                    : "Bookmark this event"}
-                </div>
-              </div>
-            );
-          })}
-        </div>
-      )),
-    [weekdays, handleEventBookmark]
+      schedule.daysEvents.map((event) => {
+        const isCurrentEventLive = isEventLive(event);
+
+        const showDate = Boolean(
+          differenceInCalendarDays(
+            eventEndTime({ event }),
+            eventStartTime({ event })
+          )
+        );
+
+        return (
+          <div key={event.id} className={CN.eventWrapper}>
+            <div className={CN.eventTitle}>
+              <span>{event.name}</span>
+              <FontAwesomeIcon
+                icon={event.isSaved ? solidBookmark : regularBookmark}
+                onClick={() => bookmarkEvent(event)}
+                className={CN.bookmarkIcon}
+              />
+            </div>
+            <div>
+              <span>
+                {isCurrentEventLive
+                  ? "Live "
+                  : formatTimeLocalised(eventStartTime({ event })) + " - "}
+              </span>
+              <span>{isCurrentEventLive && "until "}</span>
+              <span>
+                {showDate && formatDateRelativeToNow(eventEndTime({ event }))}
+              </span>{" "}
```

Comment:
Like this it is hard to figure out by just looking at the code. Is it

```suggestion
              </span>{STRING_SPACE}
```

or is it

```suggestion
              </span>
```

?

Reply:
the first one, replaced with a const
[04e9125](https://github.com/sparkletown/sparkle/pull/2840/commits/04e91251fb1d5bfc42be24e1adab66087443d930)